ui/imports/shared/popups/CommunityMembershipSetupDialog.qml

```diff
@@ -183,27 +183,8 @@ StatusStackModal {
             }
         }
 
-        property var initialAddressesModel: SortFilterProxyModel {
+        readonly property var initialAddressesModel: SortFilterProxyModel {
             sourceModel: root.walletAccountsModel
-            sorters: [
```

Comment:
Removing the old sorting code (walletType/position/name) but keeping the SFPM; some other JS code here relies on its `.count/.get` methods